target/tasks/snapd-vendor-sync/task.yaml

```diff
@@ -41,7 +41,9 @@ execute: |
     eval `ssh-agent -s`
     mkdir -p ${HOME}/.ssh && ssh-keyscan -t rsa git.launchpad.net >> ~/.ssh/known_hosts
     ssh-agent $(ssh-add /tmp/id_rsa; git clone git+ssh://snappy-m-o@git.launchpad.net/snapd-vendor /tmp/$TARGET_DIR)
-    mkdir -p $TARGET_DIR && mv /tmp/$TARGET_DIR/.git $TARGET_DIR && rm -rf /tmp/$TARGET_DIR
+    mkdir -p $TARGET_DIR
+    mv /tmp/$TARGET_DIR/.git $TARGET_DIR
+    mv /tmp/$TARGET_DIR/commits $TARGET_DIR || true
```

Comment:
Is it OK if the _commits_ file is missing? Isn't it it going to fail at a later stage when we need commit id?

Reply:
It will miss until the branch lp-buidl-core is executed and creates that. Ideally, it will miss just once.
I'll work on some documentation to make sure the whole process is clear.
Thanks for reviewing